infrastructure/main.tf

```diff
@@ -14,6 +14,7 @@ locals {
   previewVaultName     = "${var.product}-bulk-scan"
   nonPreviewVaultName  = "${var.product}-bulk-scan-${var.env}"
   vaultName            = "${local.is_preview ? local.previewVaultName : local.nonPreviewVaultName}"
+  tokenValidity        = "${var.token_validity}"
```

Comment:
This variable copies the original value and brings no added value. Why not to use `${var.token_validity}` instead?